.github/workflows/macos.yml

```diff
@@ -25,3 +25,7 @@ jobs:
       - name: Run specs
         run: |
           bundle exec rake
+      - name: Test reverse_adoc binary
```

Comment:
As I see this is just a test, if so, why not make it as regular ruby test?

Reply:
Ok, can move it to ruby test, what about the lines with dependencies install(libreoffice)?

Reply:
Should we add libreoffice for OSX too?

Reply:
Right now i have skipped it because on macos i was not been able to run LibreOffice` soffice binary - github action build hangs.